embedding/cpp/include/flutter_app.h

```diff
@@ -52,9 +52,6 @@ class FlutterApp : public flutter::PluginRegistry {
  protected:
   void ParseEngineArgs();
 
-  // Whether the app is headed or headless.
-  bool is_headed_ = true;
```

Comment:
Is there any problem with backward-compatibility if the public API is deleted without the deprecation process?